src/app/profile/yourGiving/recipientView/recipientGift/recipientGift.tpl.html

```diff
@@ -38,8 +37,18 @@ <h4 class="mb0">{{$ctrl.recipient['designation-name']}}</h4>
               <label translate>Recurring Gifts:</label>
               <ul class="list-unstyled list-recurring">
                 <li ng-repeat="recurring in $ctrl.recipient['recurring-donations']['donations']">
-                  <h4 translate>{{recurring['donation-lines'][0].amount | currency:'$'}} {{recurring.rate.recurrence.interval}}</h4>
-                  <span class="meta" ng-if="recurring['next-draw-date']" translate>Next Gift: {{recurring['next-draw-date']['display-value'] | date:'mediumDate'}}</span>
+                  <h4 translate>{{recurring['donation-lines'][0].amount | currency:'$'}}</h4>
+                  <div ng-switch="recurring.rate.recurrence.interval">
+                    <span class="meta" ng-switch-when="Monthly" translate>
+                      Monthly on the {{recurring['recurring-day-of-month'] | ordinal}}
+                    </span>
+                    <div class="u-block" ng-switch-when="Quarterly" translate>
+                      Quarterly on the {{recurring['recurring-day-of-month'] | ordinal}}
```

Comment:
I think this should specify the months. You can use this if you don't have it elsewhere https://github.com/CruGlobal/give-web/blob/master/src/common/services/giftHelpers/giftDates.service.js#L10
It does need the next gift date though... Sounds like from your hipchat conversation, that might be harder to get at.